sonarqubescannermsbuild.groovy

```diff
@@ -37,6 +37,24 @@ for (JSONObject release : releases) {
       json << ["id": "${tagName}-netcore3".toString(),
              "name": "SonarScanner for MSBuild ${tagName} - .NET Core 3.0".toString(),
              "url": "https://github.com/SonarSource/sonar-scanner-msbuild/releases/download/${tagName}/sonar-scanner-msbuild-${tagName}-netcoreapp3.0.zip".toString()];
+
+    } else {
```

Comment:
After discussing with Costin, the expectation here is to have only 2 entries that will led to the two zip files.

Entry 1:
ID: ${tagName}-net-framework
NAME: SonarScanner for MSBuild ${tagName} - .NET Fwk
URL: https://github.com/SonarSource/sonar-scanner-msbuild/releases/download/${tagName}/sonar-scanner-${tagName}-net-framework.zip

Entry 2

ID: ${tagName}-net
NAME: SonarScanner for MSBuild ${tagName} - .NET
URL: https://github.com/SonarSource/sonar-scanner-msbuild/releases/download/${tagName}/sonar-scanner-${tagName}-net.zip

Reply:
If possible, we can change the names to be 

`SonarScanner for .NET ${tagName}`

and

`SonarScanner for .NET Framework ${tagName}`

In the documentation we use[ SonarScanner for .NET](https://docs.sonarsource.com/sonarqube/9.8/analyzing-source-code/scanners/sonarscanner-for-dotnet/)